### PR TITLE
fix Date-mocks in unit tests

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by TommyDew42** [comment]
How's this going to improve the tests?

**Comment by Uzlopak** [comment]
What seems wrong here? ;)

https://github.com/fastify/one-line-logger/blob/5bc42200982397fc4811b333a99f91502fd68442/test/helpers.js#L53

File: test/helpers.js [hunk 1]
```diff
@@ -25,47 +25,41 @@ const pinoFactory = (opts) => {
...
+  const pinoLogger = pinoFactory({ destination, ...opts })
 
   return fastify({
-    logger: pino,
+    logger: pinoLogger,
     ...fastifyOpts
   })
 }
 
+const DateOriginalNow = Date.now
```
Comment (TommyDew42) [inline test/helpers.js:43]: maybe `dateOriginalNow` or `DATE_ORIGINAL_NOW`?

Comment (Uzlopak) [reply test/helpers.js:43]: I dont think it make a big difference.

Comment (TommyDew42) [reply test/helpers.js:43]: Yes agreed that it doesn’t make a big difference!

Just trying to provide my two cents here cuz usually only namespaces or classes use an upper camel case naming in JS.

Comment (Uzlopak) [reply test/helpers.js:43]: I understand. I just found this issue when I did a sonarqube scan over the whole organization. I just wanted to fix that alert.

Comment (Uzlopak) [reply test/helpers.js]: fixed

**Comment by Uzlopak** [comment]
@mcollina 

I can not merge this PR, as it is not part of fastify/libraries or fastify/plugins team.

**Pull request merged**